src/notes/FluxNotesEditor.jsx

```diff
@@ -415,16 +415,18 @@ class FluxNotesEditor extends React.Component {
         if (Lang.isNull(selection)) {
             // Removes the shortcut from its parent
             shortcut.onBeforeDeleted();
-            return state;
         }
         let transform;
-        transform = this.state.state.transform();
-
-        shortcut.setText(selection.context);
-        if (shortcut.isContext()) {
-            shortcut.setValueObject(selection.object);
-            if (!Lang.includes(this.contextManager.contexts, shortcut)) this.contextManager.addShortcutToContext(shortcut);
-            this.contextManager.contextUpdated();
+        transform = state.transform();
+    
+        shortcut.clearValueSelectionOptions();
```

Comment:
[MG2] Line 422 was likely added back in during a rebase. We need to get rid of this line to ensure that we have the functionality that I made for re-selecting values from completion portals

Reply:
whoops!